docs/output.md

```diff
@@ -35,7 +35,7 @@ Stand-alone graphical outputs are placed in this directory. They may be useful i
     - `[coloring variable]/png/sample_dendrogram.png`: A sample clustering dendrogram
     - `[coloring variable]/png/mad_correlation.png`: Outlier prediction plots using median absolute deviation (MAD)
   - `differential/`: Directory containing standalone plots from differential analysis. Plots are stored in directories named for the associated contrast.
-    - `[contrast]/png/volcano.png`: Volcano plots of -log(10) p value agains log(2) fold changes
+    - `[contrast]/png/volcano.png`: Volcano plots of -log(10) p value against log(2) fold changes. Each dot in the plot represents one gene. The higher genes in the volcano plot are highly significant, and the genes located in right or left side of the plot are highly dysregulated.
```

Comment:
@pinin4fjords where would you rather have this?

Reply:
I don't think the additional description adds anything. The statements added follow from what was already there (if you understand what a fold change and -log10(p value) are. If this level of detail is added here we'd have to add it for all files and I think it would just make things wordier without adding additional meaning.